### PR TITLE
Dan/wrap auth

### PR DESCRIPTION
**Pull request by sabind**
Description
-----------

Tidy up Auth PR

Motivation
----------

Just tweaking a few things from @ashwinkjoseph

Testing Guidelines
------------------

Release Checklist
-----------------

- [ ] This code has unit tests
- [ ] I have a plan to verify that these changes are working as expected after deploy
- [ ] I have a plan for how to revert, rollback, or disable these changes if things are not working as expected

Security Checklist
-------------------

This PR creates, modifies, or deletes:

- [ ] API routes: API routes, parameters, or user authorization
- [ ] Authentication: Authentication mechanism
- [ ] Credentials: Server side credentials, or secrets in configuration / source code
- [ ] Cryptography: Encryption, hashing, certificates, signatures, random numbers, etc.
- [ ] User data: personal information handling, logs, error messages, etc.

Additional Notes
----------------

### CODE REVIEW

File: web-client/package.json [hunk 1]
```diff
@@ -43,7 +43,10 @@
   },
   "scripts": {
     "start": "SKIP_PREFLIGHT_CHECK=true react-app-rewired start",
-    "build": "NODE_ENV=development react-app-rewired build",
+    "build": "react-app-rewired build",
```
Comment (sabind) [inline web-client/package.json:46]: This is mostly the section I need help with in terms of using env vars in react.

`web-client/src/firebase/index.ts` contains the logic that should leverage the env var to load the proper config.

File: web-client/src/firebase/index.ts [hunk 2]
```diff
@@ -9,8 +8,8 @@ const config = {
     projectId: 'reach4help-dev',
     storageBucket: 'reach4help-dev.appspot.com',
     messagingSenderId: '664078449280',
-    appId: '1:664078449280:web:fad2c3b30771213e5f4465',
-    measurementId: 'G-WFTRYNEQBB',
+    appId: '1:664078449280:web:115f33485dfc48e35f4465',
+    measurementId: 'G-DKH4PLZ62L',
```
Comment (jpmarques66) [inline web-client/src/firebase/index.ts:12]: I think it isn't a good idea to have these keys and ids publicly in our repo 🤔

Comment (jpmarques66) [reply web-client/src/firebase/index.ts:12]: Maybe this can go into .env files, what do ya think?

Comment (jpmarques66) [reply web-client/src/firebase/index.ts:12]: I talked with Joseph and he'll handle this 😄 . Also, I don't if it is a good idea to maintain these values in the future. I think we should refresh the app id's and keys after they get removed from the code base. Maybe @mdeous can support me here.

Comment (sabind) [reply web-client/src/firebase/index.ts:12]: https://firebase.google.com/docs/hosting/reserved-urls

Comment (sabind) [reply web-client/src/firebase/index.ts:12]: > I talked with Joseph and he'll handle this 😄 . Also, I don't if it is a good idea to maintain these values in the future. I think we should refresh the app id's and keys after they get removed from the code base. Maybe @mdeous can support me here.

Will @ashwinkjoseph be taking over the work directly in this PR?

Comment (jpmarques66) [reply web-client/src/firebase/index.ts:12]: I think he'll be creating other PR.

**Pull request closed without merging**